page.go

```diff
@@ -695,6 +695,9 @@ func (p *pageImpl) TextContent(selector string, options ...FrameTextContentOptio
 }
 
 func (p *pageImpl) Video() Video {
+	p.Lock()
```

Comment:
This isn't related to the issue #253, but for race condition detected in a CI test.
https://github.com/playwright-community/playwright-go/runs/6224590861